pkg/karmadactl/cmdinit/karmada/deploy.go

```diff
@@ -178,6 +160,38 @@ func createExtralResources(clientSet *kubernetes.Clientset, dir string) error {
 	return nil
 }
 
+// crdInitialize initialize crd
+func crdInitialize(crdClient *clientset.Clientset, dir, caBase64 string) error {
+	if err := utils.PathExists(dir + "/crds/bases"); err != nil {
+		return err
+	}
+	basesCRDsFiles := utils.ListFiles(dir + "/crds/bases")
```

Comment:
Hi, have you tested if this dir is just a empty dir, if there is any other unexpected failure?